photon-client/src/types/WebsocketDataTypes.ts

```diff
@@ -46,13 +53,14 @@ export type WebsocketVideoFormat = Record<
 >;
 
 export interface WebsocketCameraSettingsUpdate {
-  calibrations: WebsocketCompleteCalib[];
```

Comment:
Can `WebsocketCompleteCalib` be safely removed? It doesn't seem to be used by anything else

Reply:
yep. And we should move towards deleting any other places you see weird misdirections that we can delete (mainly looking at places we convert objects to hashmaps in the backend manually instead of using Jackson)